binder/requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy
```

Comment:
Move the pil and ply dependencies here from the apt file. Delete numpy there. Test if the packages are there, e.g., with (in the notebook):

```
!grass --tmp-location XY --exec python -c "import matplotlib"
```

Reply:
When I try to move the pil and ply dependencies, I get an error when I try to build binder:

'''
ERROR: Could not find a version that satisfies the requirement pil
ERROR: No matching distribution found for pil
'''

From my googling, it seems like PIL has to be installed first. Should it remain in the apt file?

Reply:
What is in the apt file goes to system Python and is not available in the notebook or GRASS when I test in the notebook using the above (which is using conda in this default Binder setup). You can try to confirm it. If that holds, apt file is not useful for Python packages. And python and python-dev should not be there either (I'm not even sure how python-dev got there, maybe to compile numpy or something).

What you import as pil, is installed with its proper name PIL. The requirements file goes to this repo: https://pypi.org/project/PIL/. So try PIL.